### ISSUE TITLE
Anyway of using this to capture a drive which is in use e.g. C:\

### ISSUE THREAD

**Issue by wingers999** [jeffkl/ManagedWimgApi#37]
Hi, does anyone know a way I could use this to capture a drive that is in use e.g. my windows drive C:

I know I can boot into PE and do it, but some imaging tools can backup a live system, and I would like to be able to do it with this?

Had though I could create a shadow using something like AlphaVSS (https://github.com/alphaleonis/AlphaVSS) 

I know in Windows 10 itself I can go to Control Panel\System and Security\Back up and Restore (Windows 7) > Create a system image and it will do it live - and looks like that uses DISM to do it?

Thanks

**Comment by wingers999** [comment]
Thanks Jeff, hadn't thought of that - hopefully I can find a wrapper or API so I can use it in my app in same way I do with DISM and WIMG

**Comment by jonmiller1** [comment]
Yes, but there's a catch.  Just like any software that wants to make a backup of files in use, you have to use the Volume Shadow Service to create a snapshot, mount the snapshot, and then you capture from the mounted snapshot.  VSS isn't the funnest api to work with.
